src/clj/game/cards/hardware.clj

```diff
@@ -396,17 +396,19 @@
                  :effect (effect (trash card {:cause :ability-cost}) (lose :tag 1))}]}
 
    "Friday Chip"
-    (let [ability {:msg (msg "move 1 virus counter to " (:title target))
-                   :req (req (pos? (get-in card [:counter :virus] 0)))
-                   :choices {:req #(and (has-subtype? % "Virus")
-                                        (is-type? % "Program"))}
-                   :effect (req (add-counter state :runner card :virus -1)
-                                (add-counter state :runner target :virus 1))}]
-     {:events {:runner-turn-begins ability
-               :runner-trash {:optional
+   (let [ability {:msg (msg "move 1 virus counter to " (:title target))
+                  :req (req (pos? (get-in card [:counter :virus] 0)))
+                  :choices {:req #(and (has-subtype? % "Virus")
+                                       (is-type? % "Program"))}
+                  :effect (req (add-counter state :runner card :virus -1)
+                               (add-counter state :runner target :virus 1))}]
+     {:events {:runner-turn-begins {:req (req (pos? (count-virus-programs state)))
```

Comment:
I'm not sure this will work, since `ability` is not in the form the game engine expects (a 5 (6?) function (it's what the `req` and `effect` macros creates). 

It's better if you just add the new req clause to the `ability` map.

Reply:
Ah, okay. Good to know.